Cargo.toml

```diff
@@ -34,6 +34,7 @@ linked_list_allocator = "0.10.4"
 [target.'cfg(target_arch = "aarch64")'.dependencies]
 tock-registers = "0.8.1"
 cortex-a = "8.0.0"
+fdt = "0.1.4"
```

Comment:
I wonder if we can switch to the `vm-fdt` crate like what we are currently doing in the `cloud-hypervisor` (https://github.com/cloud-hypervisor/cloud-hypervisor/blob/1c25d71127bff42ca4caeb0a5b9a9805bdf9aac1/arch/Cargo.toml#L29) instead of this one. Using a unified fdt crate would reduce the future maintenance burden, but I do understand this is a quite big change and probably it is not reasonable to ask here.

/cc @michael2012z for discussion.

Reply:
Ah you need to parse the device tree instead of just creating stuff...my bad, sorry.

Reply:
It's better to use the same crate with Cloud Hypervisor, but we cannot use `vm-fdt` in no_std environment.